### PR TITLE
Add a DateWidget.

### PR DESCRIPTION
**Pull request by n1k0**
refs #142, r=? @leplatrem

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -312,7 +314,7 @@ export function toIdSchema(schema, id, definitions) {
   return idSchema;
 }
 
-export function parseDateString(dateString) {
+export function parseDateString(dateString, includeTime = true) {
```
Comment (leplatrem) [inline src/utils.js]: Nit: wouldn't it better to have `parseDateString()` and `parseDatetimeString()` instead ?

File: README.md [hunk 2]
```diff
@@ -217,7 +217,7 @@ Here's a list of supported alternative widgets for different JSONSchema data typ
 
 The built-in string field also supports the JSONSchema `format` property, and will render an appropriate widget by default for the following formats:
 
-- `date-time`: Six `select` elements are used to select the year, the month, the day, the hour, the minute and the second;
+- `date-time`: Six `select` elements are used to select the year, the month, the day, the hour, the minute and the second; If you don't want to deal with time, a *date* `ui:widget` is also available;
```
Comment (leplatrem) [inline README.md]: nit: `date` instead of *date* ?

**Comment by leplatrem** [comment]
Looks good to me :)

File: src/utils.js (hunk 1 above)
Comment (n1k0) [inline src/utils.js]: I tried but it was weird; we're really *always* parsing datetimes here (the JSON date format always exposes hours, minutes and seconds), and it's just an export option I'm adding here.

**Pull request merged**